src/lib/tenant/tenantUIFactory.ts

```diff
@@ -155,6 +171,7 @@ const optimismUI = new TenantUI({
   hero: optimismHero,
 
   delegates: {
+    allowed: [],
     advanced: [],
```

Comment:
Do you need to set an empty array for `ensUI`'s equivalent of a `delegates`?  Or, is optimism and ether.fi the only two that have non-null delegate-information?

Reply:
That parameter is optional, I will remove it for OP. Good catch.